cmd/conduit/api/client.go

```diff
@@ -0,0 +1,55 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"context"
+	"fmt"
+
+	apiv1 "github.com/conduitio/conduit/proto/api/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	healthgrpc "google.golang.org/grpc/health/grpc_health_v1"
+)
+
+type Client struct {
+	conn            *grpc.ClientConn
+	pipelineService apiv1.PipelineServiceClient
+	HealthService   healthgrpc.HealthClient
+}
```

Comment:
Thinking out loud - do we need a client abstraction here? Seems like it will contain just lots of boilerplate code. Can we get away with just giving the grpc client to the commands? 🤔 Or maybe, for convenience, it could just combine all service clients and make them available as public struct fields (same as `HealthService`), then we don't have to write methods that forward calls to the specific service client.

Reply:
IMHO it plays a role how we manage to inject the client into commands. Having a single client that holds all services could be handy, because it might be easier to instantiate once and inject into all commands that need it.

Reply:
@lovromazgon the idea I had was to abstract this for the CLI only in case we wanted to change specific versions on different endpoints (services) to keep these two independent. Maybe we could leave these for now here and refactor when we do really need to.

Reply:
I do see the value in having a single client that holds all services, but I fail to see the need to forward the calls through methods like `ListPipelines` in line 49. I'd just make `pipelineService` public and let the command call it directly - less abstraction, less boilerplate code.